cmd/storj-sim/network.go

```diff
@@ -271,6 +271,8 @@ func identitySetup(network *Processes) (*Processes, error) {
 		identity.Arguments = Arguments{
 			"setup": {
 				"--identity-dir", process.Directory,
+				"--concurrency", "1",
```

Comment:
Why 1?

Reply:
Since we are likely to find with a cert with one core, no need to have contention between the processes.